c_emulator/riscv_sim.c

```diff
@@ -1075,3 +1109,56 @@ int main(int argc, char **argv)
   flush_logs();
   close_logs();
 }
+
+int rv_cfg_c_ext_enable(char * isa_str, char * ext_pattern)
```

Comment:
I think it could be neat if we can call this function with ext_pattern directly from SAIL, thus each backend has to imeplemnt a single function instead of having to maintain available extensions in  multiple backends ?